docs/ci/codemagic.mdx

```diff
@@ -152,14 +152,87 @@ You can find a working example in this repository:
 
   </TabItem>
   <TabItem value="windows" label="Windows" default>
-The documentation for Windows is under development.
-<a href="https://github.com/shorebirdtech/docs/issues/98">
-Contributions welcome</a>.
+
+1. Add the group "shorebird" to your environment groups. If you're using a
+   different group name, use that instead. Also add a variable named `shorebird`
```

Comment:
```suggestion
1. Add the group containing `SHOREBIRD_TOKEN` to your environment groups. Also add a variable named `shorebird`
```

Reply:
Makes sense. Minor edit for clarity (if I'm understanding correctly).

Reply:
As context: In the section above, I used "shorebird" as group name. Should we also mention the group name "shorebird"?

![image](https://github.com/shorebirdtech/docs/assets/24459435/558c35d4-0405-47fe-99f0-774cd6f16923)

Reply:
Yes, that sounds great! Thanks for the suggestion!